app/javascript/App/index.js

```diff
@@ -19,13 +20,15 @@ import GlobalCoinSearch from './components/GlobalCoinSearch'
 import TwitterFeed from './components/TwitterFeed'
 import RedditFeed from './components/RedditFeed'
 import IcoFilters from './components/IcoFilters'
+import NewsfeedPageNew from './bundles/NewsfeedPage'
```

Comment:
is there a more semantic name for this component?

Reply:
I've removed this entirely from index.js to avoid confusion. It was only for development